airbyte-config/config-models/src/main/java/io/airbyte/config/CatalogDefinitionsConfig.java

```diff
@@ -14,13 +14,8 @@ public class CatalogDefinitionsConfig {
    * Airbyte developers to test out changes to the catalog locally.
    */
   public static String getLocalConnectorCatalogPath() {
-    Optional<String> customCatalogPath = new EnvConfigs().getLocalCatalogPath();
-    if (customCatalogPath.isPresent()) {
-      return customCatalogPath.get();
-    }
-
-    return AirbyteCatalogConstants.DEFAULT_LOCAL_CONNECTOR_CATALOG_PATH;
-
+    final Optional<String> customCatalogPath = new EnvConfigs().getLocalCatalogPath();
```

Comment:
nit: this should have used micronaut configs and specified the env var via the application.yml, instead of EnvConfigs, which will go away soon.